PowerEditor/installer/nppSetup.nsi

```diff
@@ -43,6 +43,12 @@ OutFile ".\build\npp.${APPVERSION}.Installer.arm64.exe"
 OutFile ".\build\npp.${APPVERSION}.Installer.exe"
 !endif
 
+; Sign both installer and uninstaller
+!ifdef SIGN_INSTALLER

```

Comment:
This compiler flag is passed from `PackageAll.bat`. That means if installer is created by just compiling `nppSetup.nsi` (e.g. right click on file and compile with nsis), then both installer and uninstaller will not be signed.